src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public interface FigureSupplier {
+    static String getFigure() {
```

Comment:
Return Figure instead String, and what about access modifiers?

Reply:
now its `static Figure getFigure()`